src/herbivoresAndCarnivores.test.js

```diff
@@ -34,8 +34,7 @@ describe('Animal', () => {
 
       expect(carn)
         .toHaveProperty('health', 100);
-    }
-  );
+    });
 
   it(`Herbivore should have 'hide' method which inverts 'hidden' property`,
```

Comment:
The test case for the 'hide' method should be checking if the 'hidden' property is set to true, not if it's inverted. According to the checklist, the 'hide' method should only set the property to true.